docker/docker-entrypoint.sh

```diff
@@ -1,19 +1,25 @@
-#!/bin/bash -xue
-
+#!/bin/bash -l
 # this script is executed at each startup of the container
+set -e
+cd $HOME
 
-# hack to make sure we have access to files in the jenkins home directory
-# the UID of jenkins in the container should match our UID on the host
-if [ ${UID} -ne 1500 ]
-then
-    cat /etc/passwd
-    sed -i "s/x:1500:/x:${UID}:/" /etc/passwd
+# make a local copy of sources
+sudo tar cf - -C /home arakoon | tar xf -
 
-    chown ${UID} /home/jenkins
-    chown ${UID} /home/jenkins/OPAM
-    chown ${UID} /home/jenkins/.bash* || true
-fi
+cd arakoon
 
 # finally execute the command the user requested
-command="cd arakoon && ./docker/suites.sh $@"
-sudo -E -i -u jenkins bash -l -c "$command"
+case $1 in
+  bash)
+	bash
```

Comment:
if we add "$@" here we can run arbitrary commands in the container?

Reply:
yes, we can.
the syntax would be like `./docker/run.sh ubuntu-16.04 bash -c top`
I'd rather change bash to sh, actually.